### PR TITLE
Correct labels from 'prefix' to 'interface'

### PR DESCRIPTION
**Pull request by brandomando**
Some of the descriptions were copied from the `netbox_prefix` module into the `netbox_device_interface` module. This PR is just to correct those.

### CODE REVIEW

File: docs/plugins/modules/netbox_device_interface/netbox.netbox.netbox_device_interface_module.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ Parameters
                                 <td>
                                                                                                                                                             </td>
                                                                 <td>
-                                            <div>Defines the prefix configuration</div>
+                                            <div>Defines the interface configuration</div>
```
Comment (FragmentedPacket) [inline docs/plugins/modules/netbox_device_interface/netbox.netbox.netbox_device_interface_module.rst:84]: These are actually generated from the modules themself.

Revert these changes and update within the module itself and then I typically generate new docs on each release and these changes will be made in these `.rst` files.

Comment (brandomando) [reply docs/plugins/modules/netbox_device_interface/netbox.netbox.netbox_device_interface_module.rst]: I updated the changes in the docstring in the module and reverted the changes to the docs file.
